docs/runtimes/README.md

```diff
@@ -71,17 +71,19 @@ functions:
 
 The `${...}` notation is the [syntax to use variables](https://serverless.com/framework/docs/providers/aws/guide/variables/) in `serverless.yml`. Bref provides a serverless plugin ("`./vendor/bref/bref`") that provides those variables:
 
+- `${bref:layer.php-82}`
 - `${bref:layer.php-81}`
 - `${bref:layer.php-80}`
 - `${bref:layer.php-74}`
 - `${bref:layer.php-73}`
+- `${bref:layer.php-82-fpm}`
```

Comment:
We should probably not include this, or add a note saying coming soon.

Reply:
Ah, shoot. Missed that one. I'll update after the CI run. 

How about making it:

```markdown
- `${bref:layer.php-82-fpm}` **experimental**
```

(and the php-82 layer above as well)